libraries/entities-renderer/src/RenderablePolyVoxEntityItem.cpp

```diff
@@ -198,13 +198,17 @@ bool isEdged(PolyVoxEntityItem::PolyVoxSurfaceStyle surfaceStyle) {
 
 void RenderablePolyVoxEntityItem::setVoxelData(const QByteArray& voxelData) {
     // accept compressed voxel information from the entity-server
+    bool is_updated = false;
```

Comment:
per coding standard, use camel case or just:
```suggestion
    bool changed = false;
```